keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/EnvironmentArtifactPin.kt

```diff
@@ -3,7 +3,6 @@ package com.netflix.spinnaker.keel.core.api
 import com.netflix.spinnaker.keel.api.artifacts.DeliveryArtifact
 
 data class EnvironmentArtifactPin(
-  val deliveryConfigName: String,
   val targetEnvironment: String,
   val reference: String,
```

Comment:
is this `reference` string the name given to the artifact in the delivery config? if so LMK and I will open up a GH issue for switching this API up a little bit as we don't have easy access to reference names and have been identifying versions by type + name + version

Reply:
Yes it is - let me switch it up now if it's easy

Reply:
it's not easy